index.html

```diff
@@ -5020,180 +5020,154 @@ <h3 id="playback-AudioBufferSourceNode">
-                        result of <code>playbackSignal(bufferTime)</code> as
-                        the output frame at <code>index</code>.
-                        </li>
-                        <li>Else, emit silence as the output frame at
-                        <code>index</code>.
-                        </li>
-                        <li>Set <code>bufferTime</code> to <code>bufferTime +
-                        (dt * computedPlaybackRate)</code>.
-                        </li>
-                      </ol>
-                    </li>
-                    <li>Increase <code>index</code> by 1.
-                    </li>
-                    <li>Increase <code>currentTime</code> by <code>dt</code>.
-                    </li>
-                  </ol>
-                </li>
-                <li>If <code>currentTime</code> &gt;= <code>stop</code>,
-                consider playback for the node to have ended. No further render
-                quanta for the node will occur.
-                </li>
-              </ol>
-            </li>
-          </ol>
+          <pre>
+// The AudioBuffer and context employed by this node
+var buffer;
+var context;
+
+// The following variables capture attribute and AudioParam values for the node.
+// They are updated k-rate, except for buffer.
+var loop;
+var detune;
+var loopEnd;
+var loopStart;
+var playbackRate;
+
+// State variables for the node's playback
+var start = 0;
+var offset;
+var stop = Infinity;
+
+var bufferTime = 0;
+var started = false;
+var enteredLoop = false;
+var dt;
+
+// Handle invocation of start method call
+function handleStart(when, pos, duration) {
+  if (arguments.length &gt;= 1) {
+    start = when;
+  }
+  offset = pos;
+  if (arguments.length &gt;= 3) {
+    stop = when + duration;
+  }
+}
+
+// Handle invocation of stop method call
+function handleStop(when) {
+  if (arguments.length &gt;= 1) {
+    stop = when;
+  }
+  else {
+    stop = context.currentTime;
+  }
+}
+
+// Interpolate a multi-channel signal value for some sample frame
+function playbackSignal(position) {
+  /*
+    This function provides the playback signal function for buffer, which is a
+    function that maps from a playhead position to a set of output signal
+    values, one for each output channel. If position corresponds to the
+    location of an exact sample frame in the buffer, this function returns
+    that frame. Otherwise, its return value is determined by a UA-supplied
+    algorithm that interpolates between sample frames in the neighborhood of
+    position.
+
+    If position is greater than or equal to loopEnd and there is no subsequent
+    sample frame in buffer, then interpolation should be based on the sequence
+    of subsequent frames beginning at loopStart.
+   */
+   ...
+}
+
+// Render a single frame of audio into the channel data arrays given by output
+// at the given element index. The frame value may be an array of channel signal
```

Comment:
Style nit: Use "|output|" and "|index|" when referring to the values of the parameters.  

May not want to say that "frame value may be an array" because we didn't say what `playbackSignal` returns.